packages/components/src/styles/components/stepper/nav.scss

```diff
@@ -0,0 +1,364 @@
+        .hds-stepper-indicator-step--status-#{$status} {
+          &.hds-stepper-indicator-step {
+            cursor: pointer;
+          }
+
+          .hds-stepper-indicator-step__status {
+            color: map.get(
+            $hds-stepper-indicator-step-status-props,
+            $status,
+            "text-color-hover"
+            );
+          }
+
+          .hds-stepper-indicator-step__svg-hexagon {
+            path {
+              fill: map.get(
+                $hds-stepper-indicator-step-status-props,
+                $status,
+                "fill-color-hover"
+              );
+              stroke: map.get(
+                $hds-stepper-indicator-step-status-props,
+                $status,
+                "stroke-color-hover"
+              );
+            }
+          }
+        }
+      }
+    }
+
+    .hds-stepper-nav__step-button:active,
+    .hds-stepper-nav__step-button.mock-active {
+      .hds-stepper-indicator-step--status-#{$status} {
+        &.hds-stepper-indicator-step {
+          cursor: default;
+        }
+
+        .hds-stepper-indicator-step__status {
+          color: map.get(
+            $hds-stepper-indicator-step-status-props,
+            $status,
+            "text-color-default"
+          );
+        }
+
+        .hds-stepper-indicator-step__svg-hexagon {
+          path {
+            fill: map.get(
+              $hds-stepper-indicator-step-status-props,
+              $status,
+              "fill-color-default"
+            );
+            stroke: map.get(
+              $hds-stepper-indicator-step-status-props,
+              $status,
+              "stroke-color-default"
+            );
+          }
+        }
+      }
+    }
+
+    &.hds-stepper-nav__step--interactive {
+      .hds-stepper-nav__step-button:active,
+      .hds-stepper-nav__step-button.mock-active {
+        .hds-stepper-indicator-step--status-#{$status} {
+          &.hds-stepper-indicator-step {
+            cursor: pointer;
+          }
+
+          .hds-stepper-indicator-step__status {
+            color: map.get(
+              $hds-stepper-indicator-step-status-props,
+              $status,
+              "text-color-active"
+            );
+          }
+
+          .hds-stepper-indicator-step__svg-hexagon {
+            path {
+              fill: map.get(
+                $hds-stepper-indicator-step-status-props,
+                $status,
+                "fill-color-active"
+              );
+              stroke: map.get(
+                $hds-stepper-indicator-step-status-props,
+                $status,
+                "stroke-color-active"
+              );
+            }
+          }
+        }
+      }
+    }
+  }
+}
+
+// MOBILE
+
+@media screen and (width <= 550px) {
+  .hds-stepper-nav {
+    &::before,
+    &::after {
+      content: none;
+    }
+  }
+
+  .hds-stepper-nav__list {
+    flex-direction: column;
+  }
+
+  .hds-stepper-nav__step:not(:last-of-type) {
+    margin-bottom: 16px;
```

Comment:
If you use a flex layout for `.hds-stepper-nav__list` then you can just set `gap: 16px` so you could remove this.